main/main.cpp

```diff
@@ -1369,6 +1372,8 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			editor = true;
 		} else if (arg == "-p" || arg == "--project-manager") { // starts project manager
 			project_manager = true;
+		} else if (arg == "--safe-mode") { // enables safe mode
```

Comment:
```suggestion
		} else if (arg == "--safe-mode") { // Enables safe mode.
```
(the other comments have incorrect style)

Reply:
Fixed